app/tests/spec/views/oauth_sign_up.js

```diff
@@ -155,25 +139,16 @@ define(function (require, exports, module) {
     });
 
     describe('submit', function () {
-      it('redirects to /confirm on success', function () {
+      it('delegates to view.afterSignUp', () => {
         fillOutSignUp(email, 'password', { context: view });
 
-        sinon.stub(user, 'signUpAccount').callsFake(function (account) {
```

Comment:
This test duplicated work done in the signup-mixin tests, so I tested that view.signUp was called, and will leave the other tests to check the permissions fuctionality.